docs/developer/available-plugins/dummy-credit-card.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Dummy Credit Card
+---
+
+## Key concepts
+
+Dummy Credit Card is a Saleor testing gateway. If you want to customize it, you can do it
```

Comment:
`If you want to customize it` -> Shouldn't be `to configure it` ? Not sure if the `customize` is a proper word here.